### DISCUSSION TITLE
Revert traditional chinese game terms to match what's seen in the client

### DISCUSSION THREAD

**Discussion by Itavjuk** [lonqie/SchaleDB#71] [Translations]
Right now several tchinese game terms on this site is machine translated from simplified chinese(community). 

Can this part be reverted to be the same as in game version?

Below are a few example: 
English--Schale.db--In game
Light Armor--輕型護甲--輕裝甲
Normal(attack type)--常規--一般
Normal(armor type)--常規護甲--一般裝甲

Many skill descriptions are also different from the client. But the community version often contains more information, so this part can stay as it is.(except a few errors made by machine translate, like "后" most time should be "後")

**Comment by factistel** [comment]
> Right now several tchinese game terms on this site is machine translated from simplified chinese(community).
> 
> Can this part be reverted to be the same as in game version?
> 
> Below are a few example: English--Schale.db--In game Light Armor--輕型護甲--輕裝甲 Normal(attack type)--常規--一般 Normal(armor type)--常規護甲--一般裝甲
> 
> Many skill descriptions are also different from the client. But the community version often contains more information, so this part can stay as it is.(except a few errors made by machine translate, like "后" most time should be "後")

這裡是目前負責中文翻譯的factistel，感謝你提出的建議。
（下面是正文）

_天使和惡魔在我的耳旁低語。
天使說
「這可是官方繁體中文！官方繁體中文的語言選項！你真的要選擇術語翻譯和簡中(社群)的保持一致嗎？」
惡魔說
「噢天哪，你也不看看官方繁中翻譯的術語是什麼東西（至少說這段話的惡魔認為，官方繁中的術語翻譯雖說並沒有什麼離譜的錯誤，但也並不能讓惡魔滿意，因為惡魔對術語這方面的要求有很高的標準，和技能描述的標準一樣高）！為什麼還要改回去！」_

借上面這段話來說明我的想法就是，兩邊都有理，我難以抉擇。
但是之前在上手做繁中UI的時候，惡魔略勝天使一籌，所以現在看到的是和簡中(社群)一致的術語翻譯。

錯字大概是寫指令碼轉換繁中的時候指令碼沒寫好，轉換完之後檢查的時候也沒檢查到導致的，之後我會盡量手動去修改一下。

**Reply by Itavjuk** [reply]
哈哈，很有趣的回覆，我知道了。
當然不是看不懂，只是突然發現有些詞不一樣。再從一些用語差異發現是從簡中來的。
就技能描述的部分，我文中也提到網站上比遊戲內好很多。以前還得特地換英文看確切數據。
繁簡轉換錯字部分，就再麻煩您了，也謝謝您的努力。
(目前看到的:
1.黑白和薇娜技能內的直"綫" 2.KAITEN內的"后" 3.泉的EX內的持"续")

**Reply by factistel** [reply]
瞭解了，在下次日服更新的時候會盡量修改已找到的錯字。

以及順便提一下，角色攻擊屬性中的「爆炸」和「貫通(貫穿)」能很明顯地看出與軍事用語中的「高爆(HE)」和「穿甲(AP)」相關，所以相比「一般」和「普通」二詞，選用了稍微更帶有一些軍事色彩的「常規」。
（寫下後面這段話只是因為在郵件提醒中看到了已經被去掉了的疑問）

**Reply by Itavjuk** [reply]
原本想說因為用繁中的不只臺灣，所以把回答修改掉了。沒想到github可以看修改紀錄，唉呀。
不過既然都提到了....(沒有要您修改的意思，只是想分享一下。)
常規在臺灣少用，真要用的話，和"慣例"同義，是一般或常見的規定/規矩的意思。
在教育上也會用做生活規範的意思。注重"規"的部分。
軍語中normal的翻譯是:正常、標準
![圖片](https://github.com/SchaleDB/SchaleDB/assets/168453386/0c1cb79d-ff7f-453e-bccc-434f0be2e06f)(來源:美華軍語辭典)
